### ISSUE TITLE
iohyve gives grup prompt after debian install

### ISSUE THREAD

**Issue by 4ad** [pr1ntf/iohyve#304]
Debian 9.8.0 installs fine:

    iohyve fetchiso https://cdimage.debian.org/debian-cd/current/amd64/iso-cd/debian-9.8.0-amd64-netinst.iso

    iohyve create debian 100G
    iohyve set debian ram=8G os=d8lvm loader=grub-bhyve

    iohyve install debian debian-9.8.0-amd64-netinst.iso
    iohyve console debian

But it won't start:

    iohyve start debian
    iohyve console debian

gives me a `grub>` prompt.

This is iohyve 0.7.9 installed from packages on FreeBSD 12.0-RELEASE-p3.

In the grub console there is `(hd0,msdos1)/grub/grub.cfg` installed by the Debian installer, of course.

On FreeBSD, in `/iohyve/debian` theres `device.map`, but no `grub.cfg`.

**Comment by 4ad** [comment]
Non-LVM install and `os=debian` works. Just the LVM install is broken.

**Comment by yo000** [comment]
late, but you can use os=ubuntu for Debian-like LVM installs
